webapi/src/test/scala/org/knora/webapi/slice/common/KnoraApiValueModelSpec.scala

```diff
@@ -58,17 +111,520 @@ object KnoraApiValueModelSpec extends ZIOSpecDefault {
     test("getResourceIri should get the id") {
       check(Gen.fromIterable(Seq(createIntegerValue, createLinkValue).map(_.toJsonPretty))) { json =>
         for {
-          model <- KnoraApiValueModel.fromJsonLd(json)
-        } yield assertTrue(model.rootResourceIri.toString == "http://rdfh.ch/0001/a-thing")
+          model <- KnoraApiCreateValueModel.fromJsonLd(json)
+        } yield assertTrue(model.resourceIri.toString == "http://rdfh.ch/0001/a-thing")
       }
     },
     test("rootResourceClassIri should get the rdfs:type") {
       check(Gen.fromIterable(Seq(createIntegerValue, createLinkValue).map(_.toJsonPretty))) { json =>
         for {
-          model            <- KnoraApiValueModel.fromJsonLd(json)
-          resourceClassIri <- model.rootResourceClassIri
-        } yield assertTrue(resourceClassIri.toString == "http://0.0.0.0:3333/ontology/0001/anything/v2#Thing")
+          model <- KnoraApiCreateValueModel.fromJsonLd(json)
+        } yield assertTrue(model.resourceClassIri.toString == "http://0.0.0.0:3333/ontology/0001/anything/v2#Thing")
       }
     },
-  ).provideSome[Scope](IriConverter.layer, StringFormatter.test)
+    test("valueNode properties should be present") {
```

Comment:
remark (non-blocking): I'm not really clear what is meant by "valueNode". Probably it's just me not being deeply enough in the matter, but maybe you find a more descriptive phrasing? Or you can explain it to me? :)

Reply:
I will clarify the title of this test in the other PR. `valueNode` does not have a special meaning anymore the anyway.

Reply:
Here is the clarification: https://github.com/dasch-swiss/dsp-api/pull/3413/commits/035efb606f2b1e12d7f755f616d7fcb0d1ca184a

Reply:
thanks!